src/new/cli.ts

```diff
@@ -30,13 +30,29 @@ const createProject = (): CommandModule<CommandModuleArgs, any> => {
 				type: "string",
 				alias: "d",
 			})
+			.option("experimental", {
+				describe: "Use experimental traspile with swc",
+				type: "boolean",
+				default: false,
+			})
 			.implies("package-manager", "template")
 			.implies("template", "package-manager")
 
 			return yargs;
 		},
-		handler: async ({projectName, packageManager , template, directory}) => {
-			return await projectForm(projectName, [packageManager, template, directory]);
+		handler: async ({
+			projectName,
```

Comment:
In GitHub review this lines are strange with some extra space, double check your LF. Running prettier in the whole project perhaps fix it

Reply:
I can't find this bad eol error, running prettier with 120 cols length fit in one line, I think it solves. Lemme know if error stills.

Reply:
Alright then, perhaps it was just a visual bug in GitHub.